src/index.html

```diff
@@ -5,9 +5,338 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="/images/favicon.png">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;1,400;1,500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
+    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header">
+      <a href="tel:+12345555555" class="header__phone">+1 234 555-55-55</a>
+      <img
+        src="/images/logo.png"
+        alt="crown logo"
+        class="header__logo"
+      >
+      <a href="tel:+12345555555" class="header__call">call to order</a>
+
+      <nav class="header__mobile nav-mobile">
+        <label for="check" class="nav-mobile__checkbtn">
+          <img
+            src="./images/burger_white.svg"
+            alt="menu icon"
+            tabindex="0">
+        </label>
+        <input
+          type="checkbox"
+          id="check"
+          class="nav-mobile__toggler"
+        >
+        <ul class="nav-mobile__list">
+          <li class="nav-mobile__item"><a href="#" class="nav-mobile__link">home</a></li>
+          <li class="nav-mobile__item"><a href="#about" class="nav-mobile__link">about us</a></li>
```

Comment:
Посмотри что писал Дима в группе) Форматируй код так чтоб было приятно читать
```suggestion
          <li class="nav-mobile__item">
            <a href="#about" class="nav-mobile__link">about us</a>
          </li>
```

Reply:
fixed